products/motorola.md

```diff
@@ -0,0 +1,429 @@
+---
+title: Motorola Mobility
+category: device
+tags: mobile-phone
+iconSlug: motorola
+permalink: /motorola
+alternate_urls:
+-   /motorolamobility
+-   /motorola-mobility
+releasePolicyLink: https://en-us.support.motorola.com/app/software-security-update
+releaseColumn: false
+releaseDateColumn: true
+activeSupportColumn: false
+eolColumn: Security Updates
+
+# This data is parsed from the HTML CDATA on the Motorola website:
+# https://en-us.support.motorola.com/app/software-security-update
+#
+# See this Gist for a parsing script:
+#     https://gist.github.com/ab/67c6343530c6b8e599f07cedccbcba3d
+#
+
+releases:
```

Comment:
Releases must be ordered by their `releaseDate` (for example this not the case for _moto e13_).

Not an exact science, but this usually moves EOL products at the bottom of the table, so they are hidden by default (such as on https://endoflife.date/python).